core/templates/dev/head/components/summary_tile/collection_summary_tile_directive.html

```diff
@@ -1,12 +1,21 @@
-<md-card class="oppia-activity-summary-tile">
+<learner-dashboard-icons ng-if="showLearnerDashboardIcons()"
+                         activity-type="ACTIVITY_TYPE_COLLECTION"
+                         activity-id="getCollectionId()"
+                         activity-title="getCollectionTitle()"
+                         activity-active="collectionIsActive">
+</learner-dashboard-icons>
+<md-card class="oppia-activity-summary-tile"
+         ng-class="{'oppia-activity-playlist-tile': isPlaylistMode()}"
+         ng-mouseenter="toggleCollectionIsActive()" ng-mouseleave="toggleCollectionIsActive()"
```

Comment:
Here and in the exp summary tile: instead of toggling, consider having setHoverState(true) and setHoverState(false). That's more explicit/definite, and avoids the assumption of no race conditions between mouseenter and mouseleave.

Reply:
Done!